discovery-provider/plugins/pedalboard/apps/relay/src/types/relay.ts

```diff
@@ -0,0 +1,23 @@
+import { Static, Type } from "@sinclair/typebox";
+
+export const RelayRequest = Type.Object({
+  senderAddress: Type.String(),
+  encodedABI: Type.String(),
+  gasLimit: Type.Integer(),
+});
+
+export type RelayRequestType = Static<typeof RelayRequest>;
+
+export const RelayReceipt = Type.Object({
+  // need to find a way to enforce actual TransactionReceipt here
+  blockHash: Type.String(),
+  blockNumber: Type.Integer(),
+});
```

Comment:
couldn't find a way to infer the validation output from web3s actual type in a reasonable amount of time so I just returned here what it looks like SDK actually uses.